--- bird-1.2.5/configure.in~	2010-08-03 17:44:33.000000000 +0200
+++ bird-1.2.5/configure.in	2011-02-17 17:59:49.058953687 +0100
@@ -207,9 +207,10 @@
 if test "$enable_client" = yes ; then
 	CLIENT=client
 	AC_CHECK_LIB(history, add_history, CLIENT_LIBS="-lhistory")
-	AC_CHECK_LIB(ncurses, tgetent, USE_TERMCAP_LIB=-lncurses,
-		AC_CHECK_LIB(curses, tgetent, USE_TERMCAP_LIB=-lcurses,
-			AC_CHECK_LIB(termcap, tgetent, USE_TERMCAP_LIB=-ltermcap)))
+	AC_CHECK_LIB(tinfow, tgetent, USE_TERMCAP_LIB=-ltinfow,
+		AC_CHECK_LIB(ncurses, tgetent, USE_TERMCAP_LIB=-lncurses,
+			AC_CHECK_LIB(curses, tgetent, USE_TERMCAP_LIB=-lcurses,
+				AC_CHECK_LIB(termcap, tgetent, USE_TERMCAP_LIB=-ltermcap))))
 	AC_CHECK_LIB(readline, rl_callback_read_char, CLIENT_LIBS="-lreadline $CLIENT_LIBS $USE_TERMCAP_LIB",
 		AC_MSG_ERROR([[The client requires GNU readline library 2.1 or newer. Either install the library or use --disable-client to compile without the client.]]), $USE_TERMCAP_LIB)
 	AC_CHECK_LIB(readline, rl_crlf, AC_DEFINE(HAVE_RL_CRLF),,$USE_TERMCAP_LIB)
