diff -ru bird-1.3.0.orig/sysdep/linux/netlink/netlink.c bird-1.3.0/sysdep/linux/netlink/netlink.c
--- bird-1.3.0.orig/sysdep/linux/netlink/netlink.c	2011-03-31 10:29:42.000000000 +0200
+++ bird-1.3.0/sysdep/linux/netlink/netlink.c	2011-04-10 00:51:11.000000000 +0200
@@ -851,8 +851,9 @@
 	   * for their 'alien' routes.
 	   */
 
-	  if (i->rtm_protocol == RTPROT_BOOT)
+	/*  if (i->rtm_protocol == RTPROT_BOOT)
 	    src = KRT_SRC_KERNEL;
+	 */
 	}
 
       break;
