--- bird-1.0.7/conf/Makefile.orig	2003-08-14 10:13:15.000000000 +0200
+++ bird-1.0.7/conf/Makefile	2004-03-28 18:21:59.429195616 +0200
@@ -24,6 +24,6 @@
 	$(M4) -P $(conf-src)/gen_commands.m4 $(srcdir)/client/cmds.m4 $(conf-fragments) | sort >commands.h
 
 cf-lex.c: cf-lex.l
-	$(FLEX) $(FLEX_DEBUG) -sB8 -ocf-lex.c -Pcf_ cf-lex.l
+	$(FLEX) $(FLEX_DEBUG) -s -B -8 -ocf-lex.c -Pcf_ cf-lex.l
 
 depend: keywords.h commands.h cf-parse.tab.c cf-lex.c
